esridump/dumper.py

```diff
@@ -415,16 +421,35 @@ def __iter__(self):
 
         query_url = self._build_url('/query')
         headers = self._build_headers()
-        for query_args in page_args:
-            try:
-                response = self._request('POST', query_url, headers=headers, data=query_args)
-                data = self._handle_esri_errors(response, "Could not retrieve this chunk of objects")
-            except socket.timeout as e:
-                raise EsriDownloadError("Timeout when connecting to URL", e)
-            except ValueError as e:
-                raise EsriDownloadError("Could not parse JSON", e)
-            except Exception as e:
-                raise EsriDownloadError("Could not connect to URL", e)
+        for query_index, query_args in enumerate(page_args, 1):
```

Comment:
Make it clearer what the `1` argument is for.

```suggestion
        for query_index, query_args in enumerate(page_args, start=1):
```